advocacy_docs/partner_docs/PreciselyConnectCDC/04-Configuratingpreciselyconnectcdc.mdx

```diff
@@ -0,0 +1,41 @@
+---
+title: 'Configuration'
+description: 'Walkthrough on configuring the integration'
+---
+
```

Comment:
For this section re-write:

## Prerequisites

Implementing Precisely Connect CDC with EDB Postgres Advanced Server requires the following components:
!!! Note
    The EDB Postgres Advanced Server, EDB Postgres Extended Server and PostgreSQL Server products will be referred to as Postgres Distribution. The specific Distribution type will be dependent upon customer need or preference.

- Precisely supported source and target systems.   The source, target or both can be your Postgres Distribution(s).  In our example we will be using EDB Postgres Advanced Server for both the source and target database.
- Precisely Connect CDC software.
- PostgreSQL management tool such as pgAdmin or Postgres Enterprise Manager.  In our example we will use pgAdmin.  This tool will be used to verify that the data is being replicated as expected.

Reply:
Guide updated.